### DISCUSSION TITLE
第3节课笔记(5班)

### DISCUSSION THREAD

**Discussion by sanbuphy** [InternLM/Tutorial#193] [笔记]
这里存放你的笔记~

**Comment by angellyao** [comment]
https://zhuanlan.zhihu.com/p/676478743

**Comment by shengmouget** [comment]
[第三课笔记](https://blog.csdn.net/shengweiit/article/details/135493928?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135493928%22%2C%22source%22%3A%22shengweiit%22%7D)

**Comment by JayceNing** [comment]
https://zhuanlan.zhihu.com/p/677115774

**Comment by GitEasonXu** [comment]
第三节笔记：https://github.com/GitEasonXu/InternLM/blob/main/homework/course_3/coursenote.md

**Comment by preordinary** [comment]
https://blog.csdn.net/preor/article/details/135512627?spm=1001.2014.3001.5501

**Comment by ATang0729** [comment]
https://github.com/ATang0729/InternLMCamp_Notes/blob/main/notes/class3-note/note_clss3.md

**Comment by ZhaoHongSen0726** [comment]
https://blog.csdn.net/m0_75085274/article/details/135514695?spm=1001.2014.3001.5502

**Comment by DreamBinary** [comment]
https://github.com/DreamBinary/InternLMCamp/tree/master/day03/note

**Comment by Farewell-CK** [comment]
https://github.com/8baby8/Farewell_Learn_puyu/blob/main/%E7%AC%AC%E4%B8%89%E8%8A%82%E8%AF%BE/note.md

**Comment by Milkve** [comment]
[第三课笔记](https://juejin.cn/spost/7323055938067742771)

**Reply by Milkve** [reply]
打不开的话看这个，https://juejin.cn/post/7323055938067742771
